sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/parser/DDLParserSuite.scala

```diff
@@ -2003,6 +2002,17 @@ class DDLParserSuite extends AnalysisTest {
     intercept(sql2, "Found duplicate clauses: TBLPROPERTIES")
   }
 
+  test("create temporary view with properties not allowed") {
```

Comment:
It would be great if we can have a JIRA ID prefix like `SPARK-32374: `.

Reply:
updated. thanks!